.chronus/changes/witemple-msft-visibility-remove-deprecated-2025-2-4-15-20-47.md

```diff
@@ -0,0 +1,29 @@
+---
+changeKind: breaking
+packages:
+  - "@typespec/compiler"
+---
+
+Removed deprecated legacy visibility APIs and converted all warnings for using string-based visibility modifiers to errors.
+
+The removed APIs include:
+
+- `getVisibility`: use `getVisibilityForClass` instead.
+- `getParameterVisibility`: use `getParameterVisibilityFilter` instead.
+- `getReturnTypeVisibility`: use `getReturnTypeVisibilityFilter` instead.
+
+Furthermore, the legacy signature of `isVisible` that accepts an array of strings has been removed. Please use the new signature that accepts `EnumMember` instead.
+
+The changed decorators include:
```

Comment:
can we also reinclude the migration example from last release just so people that still haven't migrated don't need to go fishing for information

Reply:
Done.